### DISCUSSION TITLE
[Feature] songs from my storage

### DISCUSSION THREAD

**Discussion by fallsdevil** [gokadzev/Musify#402] [Ideas]
### Is there an existing issue for this?

- [X] I have searched the existing issues and confirmed that this feature is not already requested.

### Category of the Feature

Unable to categorize

### Credits

me 

### Pain

So, I have a lot of songs saved in my storage and I would also like to listen to them in this same app, as I would have to use two different apps for each function

### Suggested Solution

possibility of accessing the audio media on my device in addition to the current functions.

**Comment by elmataviejas5469-a11y** [comment]
Why put this in the discussion area?? I think this should be added also
